src/com/mateacademy/concurrency/ScheduledFutureApp.java

```diff
@@ -0,0 +1,33 @@
+package com.mateacademy.concurrency;
+
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
+
+public class ScheduledFutureApp extends Thread{
```

Comment:
dont mix your app class with method main() which you use to launch program with class with logic
dont make app class extended from Thread